pkg/cmd/infra/router/template.go

```diff
@@ -287,6 +294,77 @@ func parseCaptureHeaders(in string) ([]templateplugin.CaptureHTTPHeader, error)
 	return captureHeaders, nil
 }
 
+func sanitizeHeaderValue(headerValue string) string {
+	santizeHeaderValue := strings.ReplaceAll(headerValue, "'", `'\''`)
+	return "'" + santizeHeaderValue + "'"
+}
+
+func parseHeadersToBeSetOrDeleted(in string) ([]templateplugin.HTTPHeader, error) {
+	var captureHeaders []templateplugin.HTTPHeader
+	var capture templateplugin.HTTPHeader
+	/*encodedIn, err := b64.StdEncoding.DecodeString(in)
+	if err != nil {
+		return captureHeaders, fmt.Errorf("failed to decode base64 encoding1: %v", encodedIn)
+	}*/
```

Comment:
Remove the commented section if not needed.

Reply:
sorry my bad